extensions/binding/org.eclipse.smarthome.binding.weatherunderground/OSGI-INF/WeatherUndergroundDiscoveryService.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
> I am not refactoring the binding lol just providing a new simple feature.

Yes, and new features should use the current guidelines. No worries, you will like it as everything becomes much easier and less error-prone! See https://github.com/eclipse/smarthome/pull/4002.